src/Tribe/Status/Manager.php

```diff
@@ -219,4 +225,34 @@ public function return_statuses_by_action( $action, $commerce ) {
 
 	}
 
+	/**
+	 * Return an array of Statuses for a Commerce with label and stock attributes
+	 *
+	 * @since TBD
+	 *
+	 * @param $commerce string a string of the Commerce System to get statuses from
+	 *
+	 * @return array an array of statues with label and stock attributes
+	 */
+	public function return_status_options( $commerce ) {
```

Comment:
While on this topic, perhaps changing the other methods to `get_` rather than `return_` as well. I wish I would have caught that before...but I didn't!